packages/platform-test-suite/bin/test.sh

```diff
@@ -78,6 +78,8 @@ case ${i} in
     ;;
     -t=*|--timeout=*)
     timeout="${i#*=}"
+    --faucet-storage-dir=*)
```

Comment:
`faucet-wallet-storage-dir` for consistency with `FAUCET_WALLET_USE_STORAGE` (should we have an option for this too?)